client/src/components/Header/Header.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled from "@emotion/styled";
+import logoSrc from "./logo.svg";
+import menuSrc from "./menu.svg";
+
+const Header = () => {
+  return (
+    <Container>
+      <MenuButton>
+        <img src={menuSrc} alt="Menu icon" />
```

Comment:
I would call it menuIconSrc, because it is the source of the Icon and not the menu itself